pallets/liquidity-pools-gateway/src/lib.rs

```diff
@@ -356,16 +368,14 @@ pub mod pallet {
 		}
 
 		/// Process an inbound message.
-		#[pallet::weight(T::WeightInfo::process_msg())]
+		#[pallet::weight(T::WeightInfo::receive_message())]
 		#[pallet::call_index(5)]
-		pub fn process_msg(
+		pub fn receive_message(
```

Comment:
NOTE. It's true that now we can receive batches, but from @cdamian changes in the queue, this method will no longer process messages, just queuing, which means that it will only take some more bytes as PoV. The current defensive weights are enough for that slight increment